server/src/main/java/org/apache/druid/segment/loading/RoundRobinStorageLocationSelectorStrategy.java

```diff
@@ -32,19 +35,25 @@
  */
 public class RoundRobinStorageLocationSelectorStrategy implements StorageLocationSelectorStrategy
 {
-
-  private final List<StorageLocation> storageLocations;
+  private List<StorageLocation> storageLocations = null;
```

Comment:
The default values ​​of reference types in Java are all null, so assigning null here should be meaningless.

Reply:
Been fixed.